src/pages/community-champions.js

```diff
@@ -680,9 +683,7 @@ const CommunityChampions = () => {
     const useBannerImageWithSpace = useMedia(screenSize.upToSmall);
     return (
         <Layout>
-            <Helmet>
-                <title>Community Champions - {metadata.title}</title>
-            </Helmet>
+            <SEO title={`Community Champions - ${metadata.title}`} />
```

Comment:
👏  niceee this is much better since this page will likely be social shared around launch as well